specification/storageimportexport/resource-manager/Microsoft.ImportExport/stable/2020-08-01/storageimportexport.json

```diff
@@ -202,7 +203,8 @@
             "in": "query",
             "description": "An integer value that specifies how many jobs at most should be returned. The value cannot exceed 100.",
             "required": false,
-            "type": "integer"
+            "type": "integer",
+            "format": "int64"
```

Comment:
Hi @ChenTanyi , This change will not impact customers as we are just adding the int format to fix linter error.